packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -52,8 +52,10 @@ function InteractionLayer({
 }) {
   const [creating, setCreating] = useState(false)
   const svgContext = useContext(SVGContext)
-  const canvasRef = useRef()
-  svgContext.canvas = canvasRef.current
+  const canvasRef = useRef(null)
+  if (canvasRef.current !== null) {
+    svgContext.canvas = canvasRef.current
+  }
```

Comment:
Now that the canvas ref is fixed in Firefox, in `SingleImageCanvas`, I don't know if these lines are still needed.

Reply:
Removed in https://github.com/zooniverse/front-end-monorepo/pull/6887/commits/2d29d57284208ce837e1a6b461ade1392bba61a2.